src/mongocxx/collection.hpp

```diff
@@ -717,7 +718,7 @@ class MONGOCXX_API collection {
 
     ///
     /// Gets an index_view to the collection.
-    class index_view indexes();
+    index_view indexes();
```

Comment:
No need for an elaborated type specifier here, was probably cargo-cult'ed in over the summer.